File: goormlevel/194193/1. 소금물의 농도 구하기/소금물의 농도 구하기.cpp
#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

int main() {
	int n = 0, m = 0;
	if (!(cin >> n >> m)) return 0;

	double salt = 7.0 / 100.0 * n;

	double answer = (salt / (n + m)) * 100.0;

	answer = floor(answer * 100) / 100.0;

	cout << fixed << setprecision(2) << answer << "\n";

	return 0;
}

File: goormlevel/194193/1. 소금물의 농도 구하기/소금물의 농도 구하기.py
import sys
import math

input = sys.stdin.readline
write = sys.stdout.write

def solve():
	n, m = map(int, input().split())

	salt = (7 / 100 * n)
	
	answer = (salt / (n + m)) * 100
	answer = math.floor(answer * 100) / 100
	
	write(f"{answer:.2f}\n")

if __name__ == "__main__":
	solve()